### ISSUE TITLE
Reddit breaks some CSS when minifying

### ISSUE THREAD

**Issue by DarylPinto** [reddit/reddit#1718]
When reddit minifies custom subreddit CSS, it causes the following code to fail: `margin-left: calc(100% - 25px)`
it removes the spaces around the `-` which makes browsers such as Chrome to ignore the declaration entirely.

**Comment by 13steinj** [comment]
Yeah this is known, there are multiple open issues/PRs about this. The version of the minifier reddit uses is out of date.
